src/TagLib/Ogg/XiphComment.cs

```diff
@@ -656,22 +737,13 @@ IEnumerator IEnumerable.GetEnumerator()
 		///    langword="null" /> if no value is present.
 		/// </value>
 		/// <remarks>
-		///    This property is implemented using the "COMMENT" or
-		///    "DESCRIPTION" field, preferring "DESCRIPTION" but using
-		///    "COMMENT" if that is the field used by the comment.
+		///    This property is implemented using the "COMMENT" field.
 		/// </remarks>
 		public override string Comment {
-			get {
```

Comment:
@decriptor : There is indeed a slight API change. This happens in the Xiph implementation. 
Before, the "COMMENT" and "DESCRIPTION" were both mapped to the TagLib field `Comment`. 
Now, this obviously doesn't apply anymore as we have a separated ``Comment` and ``Description` filed. 
As a result, there is a change in behaviour, compare to previous implementations. 
However, I think this PR makes it more standard-compliant and more clear on what to expect from Comment/Description.

Reply:
I'm good with that. Lets just make sure we document that in the release notes. We should probably look at getting release out before the end of the year :)

Reply:
I think that a release for Xmas would please many people indeed ;)

Reply:
Fixed !

Reply:
Who is taking care of the release notes ? 
Shall I start updating the "NEWS" file with the user-visible changes I've done ?

Reply:
@Starwer if you'd like to do that I'd very much appreciate it